api/internal/managers/app/release/template.go

```diff
@@ -215,14 +209,55 @@ func (m *appReleaseManager) dryRunHelmChart(ctx context.Context, templatedCR *ko
 	}
 
 	// Perform dry run rendering
-	manifests, err := helmClient.Render(ctx, installOpts)
+	if err := m.setupHelmClient(); err != nil {
+		return nil, fmt.Errorf("setup helm client: %w", err)
+	}
+
+	manifests, err := m.hcli.Render(ctx, installOpts)
 	if err != nil {
 		return nil, fmt.Errorf("render helm chart %s: %w", templatedCR.Name, err)
 	}
 
 	return manifests, nil
 }
 
+func (m *appReleaseManager) setupHelmClient() error {
```

Comment:
i think we should break this template.go file down later on, it feels overloaded now